journeys/AzeezTM.md

```diff
@@ -0,0 +1,26 @@
+# <h1 align="center">[AzeezTM](https://github.com/AzeezTM)' journey</h1>
+<h2 align="center">A front-end developer from Nigeria</h2>
+
+- i am Mubarak, from Nigeria, i am a front-end developer,a Wildlife Conservationist and an Ecotourist.
+
+- i like to tour, i have travelled and visted most of the tourist sites in southwestern Nigeria.
+
+- i am currently learning some programming languages so that i will use it to build projects that will be useful in conserving wildlife and to create innovative travel software solutions.
+
+## My Journey
+- i started coding this january, 2022 where i learnt how to code at SQI College of ICT,Ibadan, Nigeria.
```

Comment:
```suggestion
- i started coding January, 2022 where i learnt how to code at SQI College of ICT,Ibadan, Nigeria.
```

don't inculde the word this because in future, after 5 years "this" will make no sense.